react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+
+export function Accordion({ array }) {
+  const [expand, setExpand] = useState(null);
+  const toggle = (i) => {
+    if (expand === i) {
+      return setExpand(null);
+    }
+    setExpand(i);
+  };
+
+  return (
+    <div className='wrapper'>
+      <div className='accordion'>
+        {array.map((item, i) => {
+          return (
+            <div key={item.number} className='item'>
+              <div className='title' onClick={() => toggle(i)}>
```

Comment:
Since you already have `item.number`, your code would be better if you use that. Then you wouldn't be dependent on the order of the items in `array`.